extensions/wikia/AdEngine/templates/recovered_message.mustache

```diff
@@ -0,0 +1,9 @@
+<div class="recovered-message {{positionClass}}">
+    <img src="/skins/oasis/images/recovered_message_icon.png" class="icon">
```

Comment:
Hm, now I wonder why we've put it there. More common way is to put an extension's image in the extension's directory. And we used to have a way, a variable which constructs the URL... `$wgUrlPath` or something in PHP. @macbre, @vforge can you help me refresh old man's memory? Or maybe there is a newer and better way to do it?

Also, let's add there `width` and `height` parameters it's supposed to save some reflows/repaints and we're up for webperf! :smile_cat: